src/firebase/actionInDatabase/addingMessagesToDatabase.ts

```diff
@@ -0,0 +1,8 @@
+import { doc, setDoc } from 'firebase/firestore'
+import { firebaseCollection, ImageMessage, TextMessage } from '../../types'
+import { db } from '../firebase'
+
+export const addingMessagesToDatabase = async (messages: ImageMessage | TextMessage, allMessages: Array<ImageMessage | TextMessage> ) => {
+    await setDoc(doc(db, firebaseCollection.rooms, messages.roomUUID), { allMessages })
```

Comment:
awaiting promise is equal to doing nothing. If you want to return it then assign result of promise to const, otherwise it won't do anything